components/eam/src/physics/cam/physics_types.F90

```diff
@@ -1742,6 +1750,9 @@ subroutine physics_state_dealloc(state)
   
   deallocate(state%pv, stat=ierr)
   if ( ierr /= 0 ) call endrun('physics_state_dealloc error: deallocation error for state%t')
+
+  deallocate(state%pt, stat=ierr)
+  if ( ierr /= 0 ) call endrun('physics_state_dealloc error: deallocation error for state%t')
```

Comment:
Message should use `state%pt` not `state%t`. Also see above, pv should be using `state%pv` but is using `state%t` as well. Please fix.

Reply:
fixed